sections/dashboard/PortfolioChart/PortfolioChart.tsx

```diff
@@ -1,29 +1,47 @@
 import { FC } from 'react';
 import styled from 'styled-components';
+import { useRecoilValue } from 'recoil';
+import useSynthetixQueries from '@synthetixio/queries';
+
 import { Synths } from 'constants/currency';
 import Currency from 'components/Currency';
 import { zeroBN } from 'utils/formatters/number';
-import Wei from '@synthetixio/wei';
+import { MobileHiddenView, MobileOnlyView } from 'components/Media';
+import useGetCurrentPortfolioValue from 'queries/futures/useGetCurrentPortfolioValue';
+import { walletAddressState } from 'store/wallet';
+import Connector from 'containers/Connector';
+import { getMarketKey } from 'utils/futures';
+import useGetFuturesMarkets from 'queries/futures/useGetFuturesMarkets';
 
-type PortfolioChartProps = {
-	totalFuturesPortfolioValue: Wei;
-	totalSpotBalanceValue: Wei;
-	totalShortsValue: Wei;
-};
+const PortfolioChart: FC = () => {
+	const futuresMarketsQuery = useGetFuturesMarkets();
+	const futuresMarkets = futuresMarketsQuery?.data ?? [];
+
+	const { network } = Connector.useContainer();
+	const markets = futuresMarkets.map(({ asset }) => getMarketKey(asset, network.id));
+	const portfolioValueQuery = useGetCurrentPortfolioValue(markets);
+	const portfolioValue = portfolioValueQuery?.data ?? null;
+
+	const walletAddress = useRecoilValue(walletAddressState);
+	const { useSynthsBalancesQuery } = useSynthetixQueries();
+	const synthsBalancesQuery = useSynthsBalancesQuery(walletAddress);
+	const synthBalances = synthsBalancesQuery.data ?? null;
+
+	const total = (portfolioValue ?? zeroBN).add(synthBalances?.totalUSDBalance ?? zeroBN);
 
-const PortfolioChart: FC<PortfolioChartProps> = (props: PortfolioChartProps) => {
-	const { totalFuturesPortfolioValue, totalSpotBalanceValue, totalShortsValue } = props;
-	const total = totalFuturesPortfolioValue.add(totalSpotBalanceValue).add(totalShortsValue);
 	return (
-		<Chart>
-			<PortfolioTitle>Portfolio Value</PortfolioTitle>
-			<PortfolioText
-				currencyKey={Synths.sUSD}
-				price={total ?? zeroBN}
-				sign={'$'}
-				conversionRate={1}
-			/>
-		</Chart>
+		<>
+			<MobileHiddenView>
+				<Chart>
+					<PortfolioTitle>Portfolio Value</PortfolioTitle>
+					<PortfolioText currencyKey={Synths.sUSD} price={total} sign="$" />
+				</Chart>
+			</MobileHiddenView>
+			<MobileOnlyView>
+				<PortfolioText currencyKey={Synths.sUSD} price={total} sign="$" />
+				<MobileChartPlaceholder />
```

Comment:
Maybe remove this for now? I'd rather see nothing in this place than a big blank spot before we get the chart implemented:
![IMG_4B3E05BD0593-1](https://user-images.githubusercontent.com/10401554/177392346-00fabf84-0aa8-4be7-97a9-625fe812f515.jpeg)

Reply:
Sure. I can take it out. Just wanted to keep it as similar to the desktop version as possible.